python/paddle/trainer_config_helpers/layers.py

```diff
@@ -225,6 +225,24 @@ def is_layer_type(type_name):
 
 
 class AggregateLevel(object):
+    """
+    As PaddlePaddle supports three sequence types:
+
+    - :code:`SequenceType.NO_SEQUENCE` means the sample is not a sequence.
+    - :code:`SequenceType.SEQUENCE` means the sample is a sequence.
+    - :code:`SequenceType.SUB_SEQUENCE` means it is a nested sequence, that 
+      each timestep of the input sequence is also a sequence.
```

Comment:
`SequenceType.SUB_SEQUENCE`  means the sample is a nested sequence, each timestep of which is also a sequence.